test/unit/lib/urlutilTest.js

```diff
@@ -179,41 +179,6 @@ describe('urlutil', function () {
     })
   })
 
-  describe('isFlashInstallUrl', function () {
```

Comment:
I don't think these tests should go away. I didn't realize these tests existed when I made the changes because the tests weren't working, but those methods can move back to urlUtil to make them unit testable again